packages/e2e-tests/src/cli/version.js

```diff
@@ -0,0 +1,47 @@
+import test from 'ava'
+import nixt from 'nixt'
+import execa from 'execa'
+
+nixt.prototype.promise = function() {
+  return new Promise((resolve,reject) => {
+    this.end(err => {
+      if(err) reject(err)
+      resolve()
+    })
+  })
+}
+
+test('should return the correct version', async t => {
```

Comment:
There are some issues with `nixt`:
- for some reason the callback style `.end()` function did not play nice with `ava` (could have been because of me), had to extend it to support promises.
- does not support snapshots - I don't think there is a way to return the `stdout` so we cannot use our own assertions.
- [does not support background tasks](https://github.com/vesln/nixt/issues/17), which is why I created the `startBackgroundProcess` utility.
- does not play nice with windows, commands need to be appended with `.cmd`, see [issue](https://github.com/nodejs/node/issues/3675#issuecomment-154264390), they should use `execa` or `cross-spawn`.

I think we should simply use `execa` and `ava` assertions instead.

Reply:
Seems a great tool anyway, but those are big downside in our case. Maybe when we test a bigger command like `aragon run` we will discover if only `execa` and `ava` is enough or we need at some point add other tooling